2355/2/README.md

```diff
@@ -0,0 +1,54 @@
+# Инструкция по применению
+
+Запустить в консоли файл __main.rb__ командой __ruby main.rb__, добавив в конце записи необходимые команды.
```

Comment:
did you read task requirements? how should I guess where to put texts of battles?